### PR TITLE
Pro 7

### PR DESCRIPTION
**Pull request by dimoiuBogdan**

### CODE REVIEW

File: src/pages/TaskDetailsPage/components/TaskDetailsSidebar/TaskDetailsSidebar.tsx [hunk 1]
```diff
@@ -1,13 +1,15 @@
 import TaskDetailsSidebarAssignees from "./TaskDetailsSidebarAssignees/TaskDetailsSidebarAssignees";
 import TaskDetailsSidebarReportingTo from "./TaskDetailsSidebarReportingTo";
-import TaskDetailsSidebarWatching from "./TaskDetailsSidebarWatching";
+import TaskDetailsSidebarWatching from "./TaskDetailsSidebarSubscribed/TaskDetailsSidebarSubscribed";
```
Comment (FratilaAndrei) [inline src/pages/TaskDetailsPage/components/TaskDetailsSidebar/TaskDetailsSidebar.tsx:3]: Please match the naming

File: src/common/data/constants.ts [hunk 2]
```diff
@@ -1,2 +1,54 @@
...
+    id: "3",
+    image:
+      "https://media.licdn.com/dms/image/D4E03AQHH85ODHCh_-Q/profile-displayphoto-shrink_800_800/0/1671533097756?e=1701907200&v=beta&t=vgXDw-_SzFvQGYzL6nnGlN6OWQk32L3_Ll4Sjsamprs",
+    name: "Iulian Curelea",
+  },
+];
+export const ALL_TAGS = [
+  {
+    id: "1",
+    name: "Frontend", // must be name for reusability
```
Comment (FratilaAndrei) [inline src/common/data/constants.ts:41]: Please remove the comment

**Review by FratilaAndrei — CHANGES_REQUESTED** [review]
Please make the required changes

File: src/common/data/constants.ts (hunk 2 above)
Comment (dimoiuBogdan) [reply src/common/data/constants.ts:41]: I'll leave it as it is important for when discussing models with the backend, to keep this in mind and not negotiate it

File: src/pages/TaskDetailsPage/components/TaskDetailsSidebar/TaskDetailsSidebar.tsx (hunk 1 above)
Comment (dimoiuBogdan) [reply src/pages/TaskDetailsPage/components/TaskDetailsSidebar/TaskDetailsSidebar.tsx:3]: Done siiiiir

**Review by FratilaAndrei — APPROVED** [review]

**Pull request merged**